boba-community/README.md

```diff
@@ -0,0 +1,77 @@
+# Running a replica node
+
+## Docker configuration
+
+Here are instructions if you want to run boba erigon version as the replica node for OP Mainnet or Testnet.
+
+### Get the data dir
+
+The first step is to download the initial data for `op-erigon`. Thanks for the contribution from [Test in Prod](https://www.testinprod.io).
+
+1. Download the correct data directory snapshot.
+
+* [OP Mainnet](https://op-erigon-backup.mainnet.testinprod.io)
+* [OP Goerli](https://op-erigon-backup.goerli.testinprod.io)
+
+2. Create the data directory and fill it.
+
+  ```bash
+  cd op-anchorage/boba-community
+  mkdir op-db
+  cd ./op-db
+  tar xvf ~/[DIR]/op-erigon-goerli.tar
+  ```
+
+3. Create a shared secret (JWT token)
+
+  ```bash
+  cd op-anchorage/boba-community
+  openssl rand -hex 32 > jwt-secrete.txt
```

Comment:
Should be "secret" rather than "secrete"?

Reply:
Good catch